### ISSUE TITLE
Chrome - focus outline 

### ISSUE THREAD

**Issue by xinyil** [Princeton-CDH/derrida-django#156]
<img width="74" alt="screen shot 2017-11-09 at 3 15 40 pm" src="https://user-images.githubusercontent.com/17734456/32739377-9f1c037c-c86d-11e7-9b3b-e3552475bef8.png">

I'm getting a focus box after mouse clicking on navigation and item titles. It should only be visible when interacting the site via keyboard tab.

**Comment by kevinglover** [comment]
@xinyil This isn't an easy ask. The outline is the active state that is applied after a link or tab press, and that behavior can vary between browsers. Here's a pretty extensive article on the issue: http://kizu.ru/en/blog/keyboard-only-focus/

**Comment by kevinglover** [comment]
The blue outline is visible for accessibility reasons. When you click on the element, the keyboard focus will also move to that element.

**Comment by xinyil** [comment]
Yes for accessibility.
Element should have the focus outline when it is TABBED; when it is CLICKED there shouldn't be a focus outline.

But it's one of those not crucial issues, so it's ok if we don't have time to fix. @rlskoeser could weigh in if we'd improve on this?

another possible reference:
https://stackoverflow.com/questions/32046375/why-pseudo-element-focus-on-button-when-using-tab-but-not-click

**Comment by rlskoeser** [comment]
If I'm understanding correctly, the browser treats these as the same thing (both trigger `focus` state) and there's no easy or reliable way to differentiate the styles for tabbed focus vs click focus.  (Thanks for the links to help clarify this.)

I don't think we should spend more time on it at this point.
